docs/sensors/i2c-sensor-addressing.md

```diff
@@ -34,7 +34,7 @@ of the `i2c-tools` package to work around this.
 | 0x06/0x07 | __0x03__ (3) | mindsensors.com Motor Multiplexer <br /> *I2C spec: Reserved for future purposes* |
 | 0x08/0x09 | __0x04__ (4) | *I2C spec: Hs-mode master code* |
 | 0x0A/0x0B | __0x05__ (5) | *I2C spec: Hs-mode master code* |
-| 0x0C/0x0D | __0x06__ (6) | *I2C spec: Hs-mode master code* |
+| 0x0C/0x0D | __0x06__ (6) | mindensors.com PPS58-Nx pressure sensor <br /> *I2C spec: Hs-mode master code* |
```

Comment:
This is the wrong line. It goes on the line that starts `| 0x18/0x19 | __0x0C__ (12) |